plugins/eddn.py

```diff
@@ -959,15 +976,20 @@ def export_journal_codexentry(  # noqa: CCR001
                            f' "{this.status_body_name}" ({type(this.status_body_name)})')
 
         else:
-            entry['BodyName'] = this.status_body_name
-            # Only set BodyID if journal BodyName matches the Status.json one.
-            # This avoids binary body issues.
-            if this.status_body_name == this.body_name:
-                if this.body_id is not None and isinstance(this.body_id, int):
-                    entry['BodyID'] = this.body_id
-
-                else:
-                    logger.warning(f'this.body_id was not set properly: "{this.body_id}" ({type(this.body_id)})')
+            # In case Frontier add it in
+            if entry.get('BodyName', None) is None:
```

Comment:
```suggestion
            if 'BodyName' not in entry:
```

Reply:
Changed, and the very similar `'BodyId'` check below this.